packages/rspack-test-tools/package.json

```diff
@@ -18,8 +18,8 @@
     "build:viewer": "rspack build",
     "dev:viewer": "rspack serve",
     "dev": "tsc -b -w",
-    "test": "sh -c 'run-s \"test:* -- $*\"' sh",
-    "testu": "sh -c 'run-s \"test:* -- -u $*\"' sh",
+    "test": "pnpm run --stream /^test:.*/",
+    "testu": "pnpm run --stream /^test:.*/ -u",
```

Comment:
does this works well on windows too?

Reply:
I'm not sure, since pnpm is platform-agnostic, I assume it could.

Reply:
I think it's already run on Windows? https://github.com/web-infra-dev/rspack/actions/runs/11066114654/job/30747516820?pr=8005

Reply:
not sure if `-u` works well, you can try to use `testu` instead in CI

Reply:
`testu` could run on Windows https://github.com/web-infra-dev/rspack/actions/runs/11090959705/job/30814366848?pr=8005#step:9:123.

I manually changed snapshot, and it passed on Windows.

<img width="884" alt="Google Chrome 2024-09-29 16 43 04" src="https://github.com/user-attachments/assets/94b17de8-41b8-4612-ab1f-b295f3b1fb3a">